### ISSUE TITLE
Project indexing hangs

### ISSUE THREAD

**Issue by rubo** [nomiclabs/hardhat-vscode-feedback#14]
It hangs on indexing a Hardhat project. Please see below:

![image](https://user-images.githubusercontent.com/337518/150222469-ec5bed06-9be5-40ee-b043-d0f271d84f1f.png)

Also, Solhint warnings don't appear anymore. Not sure whether it's connected to hung indexing or something else.

**Plugin version: 0.0.21**

**Comment by alcuadrado** [comment]
Do you have a link to a repository so that we can reproduce this?

**Comment by rubo** [comment]
@alcuadrado No, unfortunately. The repo is private. However, I can share some extra info if you need it.

**Comment by BinaryFox83** [comment]
I am getting the same issue. I just created a sample project with Hardhat
![image](https://user-images.githubusercontent.com/14966614/154808025-c6e9ea09-486f-4cef-b823-b43805ca7797.png)

**Comment by akhanubis** [comment]
Same issue. Happens with all my repos (they are private)

**Comment by rubo** [comment]
As of v0.0.24, looks like this is no longer an issue.
